### ISSUE TITLE
New rule: Ban top-level querying in module.

### ISSUE THREAD

**Issue by koddsson** [github/eslint-plugin-github#228]
Querying should be wrapped in event listeners such as `document.addEventListener('DOMContentLoaded', ...)` or a Web Component.

## 👎🏻  Examples of bad code.

```js
const el = document.querySelector('.my-element')
```

## 👍🏻  Example of good code.

```js
document.addEventListener('DOMContentLoaded', function() {
  const el = document.querySelector('.my-element')
})
```
